main.go

```diff
@@ -62,10 +65,17 @@ func main() {
 				}).Dial,
 			},
 		}
-		var queueService queue = newQueueService(&messageProducer)
+		var queueService queue
+		if *proxyAddress != "" {
```

Comment:
Can you remind me of why are we keeping the existing functionality(posting directly to kafka). Also are you going to create a new service file with a different name so that we have one cp with writes to kafka and one which writes to concept-rw-s3